fontique/src/backend/fontconfig/config.rs

```diff
@@ -17,7 +17,13 @@ pub fn parse_config(path: &Path, sink: &mut impl ParserSink) {
     let Ok(text) = std::fs::read_to_string(path) else {
         return;
     };
-    let Ok(doc) = roxmltree::Document::parse(&text) else {
+    let Ok(doc) = roxmltree::Document::parse_with_options(
+        &text,
+        roxmltree::ParsingOptions {
+            allow_dtd: true,
+            nodes_limit: u32::MAX,
```

Comment:
I'm guessing we don't care about a malicious file here which is too big to parse?

Reply:
This simply retains current behavior. I'm not opposed to placing a limit here but I'm not exactly sure what it would be.